wfoverlap/default.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv, lib, fetchFromGitHub, gfortran, makeWrapper, blas-i8, lapack }:
```

Comment:
Same comment here. I think both should be `*-i8` here. Also the convention in `nixpkgs` would tend to write `blas` and `lapack` in the input parameters and then set callPackage:
```
wfoverlap = callPackage ./wfoverlap { blas=self.blas-i8; lapack=lapack-i8; };
```

Reply:
You are right. When switching to lapack-i8 it builds flawless. Changed.